### PR TITLE
Include instructions for adding tests

### PR DESCRIPTION
**Pull request by alextricity25**
Include instructions in CONTRIBUTING for how to include tests
for a newly introduced openstack service

Connects https://github.com/rcbops/rpc-openstack/issues/1800

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -251,6 +254,15 @@ For information on how to add a service to our f5-config.py script, please see h
...
+If the introduced OpenStack service has tempest tests, make sure that it is being run as part of our jenkins-rpc gating jobs. The tempest scenarios for these jobs are defined [here](https://github.com/rcbops/jenkins-rpc/blob/master/rpc-jobs/jobs.yaml#L6). If these tests don't include the new service, please make a PR against jenkins-rpc to modify this variable.
+
+##### Adding tests to jenkins-rpc<a name="adding-tests-to-jenkins-rpc"></a>
+If the OpenStack service does not have tempest tests, then it is required for some form of functional and/or integration testing be added as part of the jenkins-rpc CI/CD pipeline. For example, the holland service has a [playbook](https://github.com/rcbops/jenkins-rpc/blob/master/scripts/test_holland.yml) to perform tests [at the end](https://github.com/rcbops/jenkins-rpc/blob/master/scripts/gating_bash_lib.sh#L369) of an AIO job. A simliar playbook can be added for the newly introduced OpenStack service. Once a playbook has been created, create a function in gating_bash_lib.sh, and invoke it in the [run_tests](https://github.com/rcbops/jenkins-rpc/blob/master/scripts/gating_bash_lib.sh#L369) function.
```
Comment (major) [inline CONTRIBUTING.md]: Monospace: `gating_bash_lib.sh`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -251,6 +254,15 @@ For information on how to add a service to our f5-config.py script, please see h
 #### Adding MaaS (Monitoring) plugins for new services<a name="add-maas-plugins"></a>
 For information on how to create MaaS plugins, please see: https://github.com/rcbops/rpc-openstack/blob/master/maas/plugins/README.md
 
+#### Integration tests<a name="integration-tests"></a>
+The newly introduced OpenStack project must have some form of integration tests. Whether that be through [tempest](http://docs.openstack.org/developer/tempest/) or our [jenkins-rpc](https://github.com/rcbops/jenkins-rpc/blob/master/README.md) gating jobs. Most OpenStack services already come with some scenario tests as part of the tempest test suite. However, if the newly introduced service does not, then integration tests can be added to the jenkins-rpc CI/CD pipeline to run on a per-commit basis.
```
Comment (robb-romans) [inline CONTRIBUTING.md]: A new OpenStack project must provide integration testing through either [tempest]...

Comment (robb-romans) [inline CONTRIBUTING.md]: Most OpenStack services provide scenario tests as part of the tempest test suite. If not, add integration tests to the `jenkins-rpc` CI/CD pipeline that run on each commit.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -251,6 +254,15 @@ For information on how to add a service to our f5-config.py script, please see h
...
+
+##### Tempest tests<a name="tempest-tests"></a>
+If the introduced OpenStack service has tempest tests, make sure that it is being run as part of our jenkins-rpc gating jobs. The tempest scenarios for these jobs are defined [here](https://github.com/rcbops/jenkins-rpc/blob/master/rpc-jobs/jobs.yaml#L6). If these tests don't include the new service, please make a PR against jenkins-rpc to modify this variable.
```
Comment (robb-romans) [inline CONTRIBUTING.md]: Please try to use active voice for technical writing.

Comment (robb-romans) [inline CONTRIBUTING.md]: [applies throughout]

**Pull request merged**